cypress/integration/join.spec.js

```diff
@@ -1,7 +1,7 @@
 import { validationErrorMessages } from '../../common/constants/messages';
```

Comment:
I know you asked about this, and I feel like I've misled you. For this file, while we do want to ensure that the errors render, I also still think there's value in ensuring that a user doesn't navigate past `/join` when using an incorrect password. The routing aspect is what makes it a valuable integration test.

Reply:
I see. So we want to ensure that the errors render but also that the front-end doesn't actually allow a submission regardless of errors?